theme-contract/src/main/java/com/orange/ouds/theme/tokens/type/BorderStyle.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme.tokens.type
```

Comment:
`BorderStyle` is not really a token but rather the type of a token value, thus should we keep this enum in the `tokens` package?

Reply:
I understand, but I don't know which name to choose for the package at the same level of `tokens`. `tokentypes` maybe?

Reply:
We already have `OudsColorScheme` in `com.orange.ouds.theme` which is another OUDS type, maybe we could move `BorderStyle` here? Also, should we rename it to `OudsBorderStyle`?